modules/govuk/manifests/apps/hmrc_manuals_api.pp

```diff
@@ -63,6 +63,8 @@
     has_readiness_health_check => true,
     log_format_is_json         => true,
     override_search_location   => $override_search_location,
+    read_timeout               => 300,
+    send_timeout               => 300,
```

Comment:
5 minutes seems like quite a large timeout given we cap the file size at 20mb. Would 30 seconds suffice?

Reply:
I think we added a `send_timeout` more for trial and error purposes, because we were still seeing 60s timeouts after increasing the `read_timeout` (it turned out to be the load balance).  That probably wasn't needed, so I'll try this without changing the `send_timeout` and see we see an improvement.

Reply:
I've confirmed we don't need to increase the `send_timeout`, so have reverted this PR back to only increasing the `read_timeout` and unicorn timeout.